packages/core/src/RepoSync.ts

```diff
@@ -202,255 +163,96 @@ export class RepoSync {
     }
   }
 
-  // Private methods
-
-  /**
-   * Called for each document upon initialization. Records the document's clock and advertises it.
-   * @param documentId
-   */
-  private async registerDoc(documentId: string) {
```

Comment:
All "registering" did was look up the doc's clock and store it in memory. Repo does that for us now.